ironic/drivers/modules/agent.py

```diff
@@ -813,6 +813,32 @@ def _delete_configuration_final(task, command):
         task.node.raid_config = {}
         task.node.save()
 
+    @METRICS.timer('AgentRAID.reset_idrac')
```

Comment:
We really shouldn't be making any changes to agent.py.  The cleaning steps that we're adding will be specific to our iDRAC driver.  You would only add the cleaning steps to agent.py if you were adding them for all drivers.

Reply:
I have corrected this one in latest commit.

Reply:
Driver management or bios interfaces please :)

Reply:
I have implemented cleaning step in management interface, please check my latest commit.

Reply:
Can you back out this change?  This is still showing up as a changed file in the PR.

Reply:
Done